samples/bluetooth/enocean/prj.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# General configuration
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_DEBUG=y
```

Comment:
```
config DEBUG
	bool "Build kernel with debugging enabled"
	help
	  Build a kernel suitable for debugging.  Right now, this option
	  only disables optimization [...]
```
Do you need this?

Reply:
Only to disable the FATAL_ERROR lib :) Removed.